app/views/providers/means/full_employment_details/show.html.erb

```diff
@@ -2,6 +2,8 @@
               url: providers_legal_aid_application_means_full_employment_details_path(@legal_aid_application),
               method: :patch,
               local: true) do |form| %>
+
+  <span class="govuk-caption-xl"><%= t("providers.means.client_caption") %></span>
```

Comment:
The pages I'm working on have the caption as `govuk-caption-l`
Just checking. If this page is different, we should check with the design team!

Reply:
Good spot. I've updated these to be `govuk-caption-l`